### DISCUSSION TITLE
How to exclude field from the mapping at all

### DISCUSSION THREAD

**Discussion by akimkelar** [spatie/laravel-data#323] [Q&A]
I have a data object, and I want to exclude one field from the serialization, regardless was it initialized or not. 
I expected it to be done easily with some annotation like `Exclude`. But it seems, it's not the case.
I tried to use class `Lazy`like

```
class Config extends Data
{
    #[Rule('required|bool')]
    public bool $active = false;

    #[Exclude]
    public Lazy|Model $model;

    public function __construct() {
        $this->model = Lazy::create(fn() => null);
    }
```

But it works only when `$model` was not initialized. When it was initialized with some real model, I get all the attributes of the model there.

Could you hint me please how it could be excluded forever?
Thank you.

**Comment by Nir-An** [comment]
Hi @akimkelar 

Could you try to use [except method](https://spatie.be/docs/laravel-data/v2/as-a-resource/lazy-properties#content-only-and-except) ?

```php
ConfigData::from(ConfigModel::first())->except('model')
```
Also, with this method, you cannot be limited to Lazy property.
